<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string>Digite seu email</string>
  </property>
  <widget class="QLabel" name="login_title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seja bem vindo(a)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_email">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Digite seu email</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Digite sua senha</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Senha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Acessar a conta</string>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgot_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Recuperar senha</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="create_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Criar uma nova conta</string>
   </property>
   <property name="text">
    <string>Criar </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
